google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -694,9 +696,11 @@ public BucketAccessControl getAcl(String bucket, String entity) {
   }
 
   @Override
-  public boolean deleteAcl(String bucket, String entity) {
+  public boolean deleteAcl(String bucket, String entity, Map<Option, ?> options) {
     try {
-      storage.bucketAccessControls().delete(bucket, entity).execute();
+      storage.bucketAccessControls().delete(bucket, entity)
+          .setUserProject(Option.USER_PROJECT.getString(options))
```

Comment:
Is there a way to set this in a generic way (so other options are also available). Because now map is essentially not a map of options, but a single userProject option.

Reply:
No, because this depends on the server side being able to handle those options. The options are generally explicitly set, e.g. in the implementation for HttpStorageRpc.read(from, options, position, bytes).